src/code/z_play.c

```diff
@@ -1,81 +1,1575 @@
 #include "global.h"
+#include "z64rumble.h"
 #include "z64quake.h"
 #include "z64shrink_window.h"
 #include "z64view.h"
+#include "overlays/gamestates/ovl_daytelop/z_daytelop.h"
+#include "overlays/gamestates/ovl_opening/z_opening.h"
+#include "overlays/gamestates/ovl_file_choose/z_file_choose.h"
+#include "overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h"
+
+s32 gDbgCamEnabled = false;
+u8 D_801D0D54 = false;
+
+// bss
+s16 sTransitionFillTimer;         // 0x801F6C10
+Input D_801F6C18;                 // 0x801F6C18
+FbDemoStruct sTrnsnUnk;           // 0x801F6C30
+u16* D_801F6D0C;                  // 0x801F6D0C
+s32 gTrnsnUnkState;               // 0x801F6D10
+VisMono sVisMono;                 // 0x801F6D18
+Color_RGBA8_u32 gVisMonoColor;    // 0x801F6D30
+Struct_80140E80 D_801F6D38;       // 0x801F6D38
+Struct_80140E80* D_801F6D4C;      // 0x801F6D4C
+BombersNotebook sBombersNotebook; // 0x801F6D50
+u8 sBombersNotebookOpen;          // 0x801F6DFC
+u8 sMotionBlurStatus;             // 0x801F6DFD
```

Comment:
Do we need these addresses? You can just look in `variables.txt` or the map file to get these

Reply:
https://github.com/zeldaret/mm/pull/1109#discussion_r990641404

Reply:
Since those variables are not static and their addresses are present on `variables.txt` and the expected map file then imo this is a bit unnecessary